irr/include/IrrlichtDevice.h

```diff
@@ -342,6 +343,20 @@ class IrrlichtDevice : public virtual IReferenceCounted
 	{
 		return video::isDriverSupported(driver);
 	}
+
+#if defined(_IRR_COMPILE_WITH_SDL_DEVICE_) || USE_SDL2
+	//! Get the scancode of the corresponding keycode.
+	virtual std::variant<u32, EKEY_CODE> getScancodeFromKey(const Keycode &key) const {
```

Comment:
the u32 type represent an implementation-defined code for the key, is that correct?
please document this better